docs/upgrade_db_zonemaster_backend_ver_7.0.0.md

```diff
@@ -30,3 +33,40 @@ cd $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backe
 perl patch_postgresql_db_zonemaster_backend_ver_7.0.0.pl
 ```
 
+
+## New dependencies
+
+With this new release, we are dropping MySQL to fully use MariaDB. This applies
+to the driver as weel, [`DBD::mysql`] is replace by [`DBD::MariaDB`].
+
+### FreeBSD
+
+```sh
+pkg install mariadb105-server p5-DBD-MariaDB
+```
+
+This will ask to remove `mysql57-server` and `p5-DBD-mysql`, answer `yes`.
```

Comment:
I think this should be in the installation instruction. This file is for the upgrade of the database schema. Even if you decide to reset the database you have to do the upgrade.

Reply:
> This file is for the upgrade of the database schema.

Right, I haven't seen that. Since we might need other packages for the next release, can't we rename this file to be more generic with a section for the database and another one for the dependencies?

Reply:
I made a commit to rename all upgrade files to be more generic: 2407446